ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/permit-simulation.tsx

```diff
@@ -45,6 +50,14 @@ const PermitSimulation: React.FC<{
     return undefined;
   }, [exchangeRate, value]);
 
+  const { tokenValue, tokenValueMaxPrecision } = useMemo(() => {
+    const valueBN = new BigNumber(value / Math.pow(10, tokenDecimals));
```

Comment:
This is the same logic we do in the decimals PR, perhaps an `applyDecimals` util?

Reply:
A new util for just division does may not be very useful.